README.rst

```diff
@@ -21,12 +21,47 @@ Setup
 ~~~~~
 
 Install python dependencies for acceptance testing (on a Jenkins instance
-running within a Docker container).
+running within a Docker container), as well as various Groovy helper utilities.
 
 .. code:: bash
 
 make requirements
 
+Configuring Jenkins
+-------------------
+
+Jenkins will run Groovy code placed in $JENKINS_HOME/init.groovy.d on each boot.
+
+
+Tips/Tricks
+~~~~~~~~~~~
+
+Unlike the Jenkins Script Console, Jenkins-related libraries are not auto-imported,
+so make sure you import the following into your scripts:
+
+.. code:: groovy
```

Comment:
@estute For some reason, I'm not seeing the groovy code when I open the README. Same goes for line 59